examples/Dockerfile

```diff
@@ -1,18 +1,29 @@
-FROM jboss/base-jdk:7
+FROM jboss/base-jdk:8
 
 ENV KEYCLOAK_VERSION 1.9.0.Final
 ENV JBOSS_HOME /opt/jboss/keycloak-demo/keycloak
+ENV MAVEN_VERSION 3.3.9
 
 USER root
 
-RUN yum -y install maven && yum clean all
+RUN yum -y install wget && yum clean all
+
+RUN curl -fsSL https://archive.apache.org/dist/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz | tar xzf - -C /usr/share \
```

Comment:
OK. We don't really need mvn 3.1.1+ for examples, but as it extends keycloak-parent (which it shouldn't) it ends up requiring. I also figured yum would have a newer version of mvn by now. So we can keep this as you've done it at least for now.